docs/knn/index.md

```diff
@@ -88,10 +88,13 @@ GET my-index/_search
 
 In this case, `k` is the number of neighbors you want the query to return, but you must also include the `size` option. Otherwise, you get `k` results for each shard (and each segment) rather than `k` results for the entire query. The plugin supports a maximum `k` value of 10,000.
 
-If you mix the `knn` query with other clauses, you might receive fewer than `k` results. In this example, the `post_filter` clause reduces the number of results from 2 to 1:
+
+## Mixing queries
+
+If you use the `knn` query alongside filters, you might receive fewer than `k` results. In this example, `post_filter` reduces the number of results from 2 to 1:
```

Comment:
If you use the `knn` query alongside filters **and other logical query clauses like bool, must, match**, you might receive fewer than `k` results. In this example, `post_filter` reduces the number of results from 2 to 1: